aspnet/fundamentals/application-insights.rst

```diff
@@ -1,7 +1,7 @@
 Application Insights
 ====================
 
-By `Steve Smith`
+By `Steve Smith <http://blog.falafel.com/author/steve-smith/>`
```

Comment:
You don't need to inline the URL - just add the missing underscore to pick up the link from authors.txt:

```
`Steve Smith`_
```